clinical-clusters/2018/clinical-clusters.json

```diff
@@ -721,22 +710,6 @@
       }
     ]
   },
-  {
-    "measureId": "117",
-    "submissionMethod": "claims",
-    "firstPerformanceYear": 2017,
-    "lastPerformanceYear": null,
-    "clinicalClusters": [
-      {
-        "name": "diabeticCare",
-        "measureIds": [
-          "001",
-          "117",
```

Comment:
same question here — does `117` need to be removed for clinicalClusters.measureIds arrays under 001 and 128?

Reply:
Same thing here... the requirement for the "diabeticCare" cluster: "Note: When submitting #117, it is not subject to EMA for these clinically related measures. If #1 and/or #128 is submitted, it is expected to submit quality measure #117. "